data/sql/updates/pending_db_world/rev_1735425042597130000.sql

```diff
@@ -0,0 +1,2 @@
+-- Remove HARD_RESET flag while retaining IMMUNITY_KNOCKBACK
+UPDATE `creature_template` SET `flags_extra` = `flags_extra` | 1073741824 WHERE `entry` = 36658;
```

Comment:
Now with your comment added I think that the sql query currently is not doing what you intended to do.
Right now it's only setting the knockback flag (which was probably already set before?)

But you wanted to remove another flag instead - so please update this to actually remove that other flag.

https://www.azerothcore.org/wiki/bit-and-bytes-tutorial

https://www.azerothcore.org/flag-checker/

Reply:
Sorry about that, changed to newer syntax forgetting bitwise operation